<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="GUR-BXTI-L162" name="channelnewsasia.com.Scenario1" tests="3" failures="2" timestamp="25 Aug 2021 08:16:20 GMT" time="217.632" errors="0">
  <testcase name="fetchWeather" time="36.035" classname="channelnewsasia.com.Scenario1"/>
  <testcase name="Scenario1method" time="88.351" classname="channelnewsasia.com.Scenario1">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Commentary: Singaporeâ€™s A330 tanker-transport plane offer can help in Afghanistan evacuation where every bit counts] but found [Biden aims for Aug 31 Afghanistan pullout as risk of attacks rises],
	expected [Commentary: US withdrawal from Afghanistan spells end of China’s free ride on international security] but found [Suspect Afghan evacuees put focus on European security screening]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Commentary: Singaporeâ€™s A330 tanker-transport plane offer can help in Afghanistan evacuation where every bit counts] but found [Biden aims for Aug 31 Afghanistan pullout as risk of attacks rises],
	expected [Commentary: US withdrawal from Afghanistan spells end of China’s free ride on international security] but found [Suspect Afghan evacuees put focus on European security screening]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at channelnewsasia.com.Scenario1.Scenario1method(Scenario1.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Scenario1method -->
  <testcase name="Scenario2method" time="93.246" classname="channelnewsasia.com.Scenario1">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Top Japanese adviser blasts IOC&#039;s Bach as wider COVID-19 curbs planned] but found [Football: Premier League clubs will not release players for red-list internationals]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Top Japanese adviser blasts IOC's Bach as wider COVID-19 curbs planned] but found [Football: Premier League clubs will not release players for red-list internationals]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at channelnewsasia.com.Scenario1.Scenario2method(Scenario1.java:79)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Scenario2method -->
</testsuite> <!-- channelnewsasia.com.Scenario1 -->
